pkg/resource/deploy/plan.go

```diff
@@ -104,6 +105,28 @@ func (p PlanPendingOperationsError) Error() string {
 	return "one or more operations are currently pending"
 }
 
+type resourceMap struct {
+	m sync.Map
```

Comment:
Can you omit the `m` to avoid the redundancy in the caller?

```suggestion
	sync.Map
```

Reply:
I'd rather not, as that clutters the method set of the type with weakly-typed methods.